src/components/vanilla/charts/DynamicAxisBar/DynamicAxisBar.emb.ts

```diff
@@ -0,0 +1,160 @@
+import { EmbeddedComponentMeta, Inputs, defineComponent } from '@embeddable.com/react';
+import { loadData, isMeasure, isDimension, Dimension, Measure } from '@embeddable.com/core';
+
+import Component, { Props } from './index';
```

Comment:
`./index` does not export Props. It even does not define Props type.

Reply:
Removed